_posts/2017-3-26-armory-and-hard-forks.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Armory and Hard Forks
+category: news
+date: 2017-03-26 23:30:00 -0500
+---
+
+Recently there has been much debate in the Bitcoin community regarding Segregated Witness, hard forks and soft forks, and the usage of alternative node and mining software. The Armory developers (goatpig, droark, and achow101) support Segregated Witness as in its current form, and also support Bitcoin Core. The Armory developers also oppose hard forks that may attack the original chain. Should a long-lasting hard fork that does not attack the original chain exist, we would consider implementing functionality required to allow Armory users to transact on that chain only if absolutely necessary (e.g. the transaction format for that chain is changed). Depending on how drastic the changes are, we may implement the change fully, implement a migration tool to another wallet that supports the change, or simply not support the fork at all.
+
+Although the Armory developers do not support such hard forks, the Armory wallet software does not perform any consensus checks as it relies on its connection to a local Bitcoin node. This node is typically Bitcoin Core, but anything based upon Core, including Bitcoin Unlimited, will work so long as no changes are made to the transaction or block formats. So long as both formats do not change, Armory should be compatible with the hard fork and will allow users to continue to transact on the forked network.
```

Comment:
How about

> Although the Armory developers do not support malicious hard forks and may not support a future non-malicious hard fork, ...

Basically a statement that although some hypothetical hard fork in the future is not supported, it won't necessarily cause Armory to fail.